frontend/src/components/shared/Radio/Radio.tsx

```diff
@@ -70,6 +71,7 @@ class Radio extends React.PureComponent<Props, State> {
   public render(): React.ReactNode {
     const { theme, width, help, label } = this.props
```

Comment:
Nit: I think horizontal could be included in the deconstruction of `this.props` on line 72 instead of being separate